vbr/support/vbrListMethods.m

```diff
@@ -1,15 +1,15 @@
-function vbrListMethods(single_prop)
+function VBR_list_methods(single_prop)
```

Comment:
@krishchvn one more change here before it's ready... This `VBR_list_methods` function should be in it's own file, `VBR_list_methods.m`. If you just cut out what you've got here and leave the `vbrListMethods` below then it should be in a good state. Thanks!

Reply:
Just to be **super** clear, here's what I mean. We should have two files:

* `VBR_list_methods.m` containing the `VBR_list_methods` function that you have here. 
* `vbrListMethods.m` containing the `vbrListMethods` function that you have down below.

Reply:
Hi. I'm so sorry I missed that out. 
I've done the appropriate changes.